src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequests.java

```diff
@@ -399,7 +399,7 @@ public final ITextComponent getShortDisplayString()
 
                 if (posInList >= 0)
                 {
-                    return ImmutableList.of(new TranslationTextComponent(AT, requester), new TranslationTextComponent(IN_QUEUE, posInList));
+                	return posInList == 0 ? ImmutableList.of(new TranslationTextComponent(FROM, requester), new TranslationTextComponent(IN_PROGRESS)) : ImmutableList.of(new TranslationTextComponent(FROM, requester), new TranslationTextComponent(IN_QUEUE, posInList));
```

Comment:
Here it's AT not FROM (last nitpick, I swear!)

Reply:
Wow another Copypasta error